### DISCUSSION TITLE
Distributed .vscodeignore files

### DISCUSSION THREAD

**Discussion by AArnott** [microsoft/vscode-discussions#798] [Extension Development QnA] [answered]
The `vsce package` command seems to only notice a `.vscodeignore` file at the root of the package. This is cumbersome and can lead to merge conflicts in a large extension and team. 

Can `vsce` be enhanced to support distributing such files around the repo, the way `.gitignore` can?

**Comment by gjsjohnmurray** [comment, marked as answer]
This request probably belongs at https://github.com/microsoft/vscode-vsce

**Reply by AArnott** [reply]
Filed at https://github.com/microsoft/vscode-vsce/issues/896

**Comment by r3m0t** [comment]
You can set a merge style of `union` for the file in `.gitattributes`

**Reply by AArnott** [reply]
True. Though that doesn't solve the problem of long paths in the one file when there could be just file names to ignore in the file in the same directory, so I'd still like distributed files, like .gitignore.
